easybuild/easyblocks/p/pytorch.py

```diff
@@ -407,6 +409,23 @@ def add_enable_option(name, enabled):
         # Metal only supported on IOS which likely doesn't work with EB, so disabled
         options.append('USE_METAL=0')
 
+        build_type = self.cfg.get('build_type')
+        if build_type is None:
+            build_type = 'Debug' if self.toolchain.options.get('debug', None) else 'Release'
+        else:
+            for name in ('prebuildopts', 'preinstallopts', 'custom_opts'):
+                if '-DCMAKE_BUILD_TYPE=' in self.cfg[name]:
+                    self.log.warning('CMAKE_BUILD_TYPE is set in %s. Ignoring build_type', name)
+                    build_type = None
+        if build_type:
+            if pytorch_version >= '1.2.0':
+                options.append('CMAKE_BUILD_TYPE', build_type)
```

Comment:
I am getting the following:
```
TypeError: append() takes exactly one argument (2 given)
```

```suggestion
                options.append('CMAKE_BUILD_TYPE=' + build_type)
```